controllers/articles_controller.js

```diff
@@ -26,20 +27,34 @@ const getArticles = (request, response, next) => {
 
 const getComments = (request, response, next) => {
   const id = request.params.article_id;
-  articleId(id)
-    .then(() => {
-      return allComments(id);
+  const promises = [articleId(id), allComments(id)];
+
+  Promise.all(promises)
+
+    .then((resolvedPromises) => {
+      response.status(200).send({ comments: resolvedPromises[1] });
     })
-    .then((comments) => {
-      if (!comments.length) {
-        response.status(200).send({ msg: "no comments" });
-      } else {
-        response.status(200).send({comments});
+    .catch((error) => {
+      next(error);
+    });
+};
+
+const patchArticles = (request, response, next) => {
+  const { article_id } = request.params;
+  const { inc_votes } = request.body;
+
+  insertPatchArticle(article_id, inc_votes)
+    .then((article) => {
+      if (article === undefined) {
+        return Promise.reject({ status: 404, msg: "Not found" });
```

Comment:
I would change the naming of these functions, patchArticles suggests you're patching multiple articles instead of just one, and insertPatchArticle suggests you're doing an insert as well as a patch, perhaps rename to something like 'patchArticleVotes'

Reply:
I've made the changes but now i have conflict and i dont know how to resolve it to pull it